### PR TITLE
fix: version/devmode toggle is back

### PR DESCRIPTION
**Pull request by pvinis**
This PR resolves [MOPLAT-]

### Description
Screenshot 2023-01-03 at 13 19 36
Screenshot 2023-01-03 at 13 19 30

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -682,7 +682,7 @@ SPEC CHECKSUMS:
   boost: a7c83b31436843459a1961bfd74b96033dc77234
   BVLinearGradient: 34a999fda29036898a09c6a6b728b0b4189e1a44
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
-  DoubleConversion: 5189b271737e1565bdce30deb4a08d647e3f5f54
+  DoubleConversion: 831926d9b8bf8166fd87886c4abab286c2422662
```
Comment (admbtlr) [inline ios/Podfile.lock:685]: what's with these changes? 🤔

File: Gemfile.lock [hunk 2]
```diff
@@ -298,8 +298,5 @@ DEPENDENCIES
   json
   psych
 
-RUBY VERSION
```
Comment (admbtlr) [inline Gemfile.lock:301]: where did this go?

File: ios/Podfile.lock (hunk 1 above)
Comment (pvinis) [reply ios/Podfile.lock:685]: just missing frok main. I run install:all, this should have been there already 🤷‍♂️

File: Gemfile.lock (hunk 2 above)
Comment (pvinis) [reply Gemfile.lock:301]: not sure. maybe bundler changed this? seems to be getting removed for me.

**Pull request merged**